### DISCUSSION TITLE
Mihile link skill buff before solving runes

### DISCUSSION THREAD

**Discussion by Tsukino-uwu** [tanjeffreyz/auto-maple#81] [Ideas]
//edit: everyone have 100% stance now with destiny.

Kanna and some other classes don't have 100% kb resist so this could probly be a workaround to make rune solving a bit more consistent if it takes long to solve them, could be added in a similar way to the interact/pet food in-game keybinds as the link skill is not class specific or just used right before moving towards solving a rune ?

**Comment by whytrade** [comment]
Mihile link skill is 100% uptime at level 2 and is already able to be set as an autobuff in command books for classes without 100% stance.

**Reply by Tsukino-uwu** [reply]
> 
Still think it could be nice to have for the lvl1 link as well, or just as a general thing instead of bound to the command books as all classes have to solve runes anyway

**Reply by whytrade** [reply]
You lose so much efficiency not having 100 stance permanently anyways, I don't get why you would be running the bot without 100 stance, youd lose way too many kills/hr

**Reply by Tsukino-uwu** [reply]
> 

I don't have a mihile lvled yet, also heard that they will change the link skill so it does not give knockback eventually so yee idk

**Comment by ScottBouloutian** [comment]
Thanks for the tip about leveling up the Mihile link skill.
